server.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+console.log('hello my bitches');
+
+const express = require('express');
+
+const app = express();
+
+const PORT = 3002
```

Comment:
Good work getting started and on making some edits to text/modules :) ( I saw your most recent PR but most changes for review are here in this one) 👍 

Ryan covered a bit more on CORS today in class but we will need to load the module ensure that we are using the middleware:

- Add: `const cors = require('cors');`
- Add: `app.use(cors());`

In addition, looks like we need to add:

- `require('dotenv').config();` so we can use our Node ENV variables 👍